eclass-writing/text.xml

```diff
@@ -327,6 +350,53 @@ variables are as follows:
     this tag. Must appear as the first tag in the comment block.
   </ti>
 </tr>
+<tr>
+  <ti>
+    <c>@PRE_INHERIT</c>
+  </ti>
```

Comment:
This (and all the following ones) will fit in one line:
```suggestion
  <ti><c>@PRE_INHERIT</c></ti>
```

See https://devmanual.gentoo.org/appendices/devbook-guide/index.html#internal-coding-style